public/app/plugins/datasource/prometheus/mode-prometheus.js

```diff
@@ -433,12 +433,12 @@ var PrometheusCompletions = function() {};
 exports.PrometheusCompletions = PrometheusCompletions;
 });
 
-ace.define("ace/mode/behaviour/prometheus",["require","exports","module","ace/lib/oop","ace/mode/behaviour","ace/mode/behaviour/cstyle","ace/token_iterator"], function(require, exports, module) {
+ace.define("ace/mode/behavior/prometheus",["require","exports","module","ace/lib/oop","ace/mode/behavior","ace/mode/behavior/cstyle","ace/token_iterator"], function(require, exports, module) {
```

Comment:
I think updating those imports will break.

Reply:
Ok, I will be taking this into account.